packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -589,7 +589,11 @@ const NewXostorForm = decorate([
         this.state.interfaceName = ev.target.value
       },
       onPoolChange(_, pool) {
-        this.state.disksByHost = {}
+        const disksByHost = {}
+        this.props.hostsByPoolId[pool?.id]?.forEach(host => {
+          disksByHost[host.id] = []
+        })
+        this.state.disksByHost = disksByHost
```

Comment:
After this changes, `isDisksMissing` will always return `false` and `numberOfHostsWithDisks` will always return the number of hosts in the pool. This is not what we expect for these computed.
Please add changes in order to:
- `isDisksMissing` to return `true`, if no disks has been selected (if every array are empty).
- `numberOfHostsWithDisks` to only count hosts with disks (number of array not empty).